dao/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>greencity</artifactId>
+        <groupId>Lv-427.Java</groupId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <groupId>greencity</groupId>
+    <artifactId>dao</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>dao</name>
+
+    <properties>
+        <junit-jupiter.version>5.4.2</junit-jupiter.version>
+        <mockito-junit-jupiter.version>3.3.3</mockito-junit-jupiter.version>
+        <liquibase.version>3.8.4</liquibase.version>
+        <modelmapper.version>2.3.5</modelmapper.version>
+        <jackson.version>2.10.1</jackson.version>
+        <powermock.version>2.0.4</powermock.version>
+        <mockito-core.version>3.3.3</mockito-core.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
No web here, only database.

Reply:
If I delete it, I won't have `javax-validation` in my entities

Reply:
Add what you need only. Add javax-validation.

Reply:
```
<dependency>
      <groupId>javax.annotation</groupId>
      <artifactId>javax.annotation-api</artifactId>
      <version>1.3.2</version>
    </dependency>
```
to be presize